core/kazoo_documents/src/kzd_devices.erl

```diff
@@ -487,6 +488,23 @@ owner_id(Doc, Default) ->
 set_owner_id(Doc, OwnerId) ->
     kz_json:set_value([<<"owner_id">>], OwnerId, Doc).
 
+-spec is_hotdesked(doc()) -> boolean().
+is_hotdesked(Doc) ->
+    not kz_json:is_empty(kz_json:get_value([<<"hotdesk">>, <<"users">>], Doc, kz_json:new())).
```

Comment:
We should probably add a `hotdesk` object to the devices schema, no? Something like:

```json
"hotdesk":{
    "type":"object",
    "description":"The hotdesk status of this device",
    "users":{
        "type":"object",
        "description":"The user(s) currently hotdesked into the device",
        "patternProperties":{
            "^[a-zA-Z0-9]{32}$":{
                "type":"object",
                "description":"user-specific hotdesk settings"
            }
        }
    }
}
```